docs/source/command_line.rst

```diff
@@ -350,6 +350,33 @@ definitions or calls.
     This flag reports an error whenever a function with type annotations
     calls a function defined without annotations.
 
+.. option:: --untyped-call-exception
+
+    This flag allows to selectively disable :option:`--disallow-untyped-calls`
+    for functions and methods defined in specific packages, modules, or classes.
+    Note that each exception entry acts as a prefix. For example (assuming there
+    are no type annotations for ``numpy`` available):
```

Comment:
Numpy's probably not a great example here, as they ship with a `py.typed` file these days and have pretty complete type annotations. Maybe one of `matplotlib`, `requests` or `TensorFlow` might be a better example? Third-party stubs exist for all three packages, but none of them ships with a `py.typed` file

Reply:
Any example of a popular library might become a bad example over time, maybe limit ourselves to something generic like `third_party_lib`?

Reply:
Yeah, let's use some generic name.